src/spec/solver_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec'
+require_relative '../solver_class'
+
+describe Solver do
+  before(:each) do
+    @solver = Solver.new
+  end
+
+  context '#factorial' do
+    it 'returns 720 for input 6' do
+      expect(@solver.factorial(6)).to eq 720
+    end
+  end
```

Comment:
- [ ] Your factorial tests only factor in one scenario, when the valuse passed in are all positives. What about when the values passed in are negatives or the values passed in is a zero? Kindly include tests of these two edge cases as well.

![image](https://user-images.githubusercontent.com/86318284/211061991-c056287b-cbed-4997-b68d-284e0cde15f6.png)

Reply:
Done!